### ISSUE TITLE
Implement recommended release process

### ISSUE THREAD

**Issue by HadesArchitect** [HadesArchitect/GrafanaCassandraDatasource#78]
https://github.com/grafana/plugin-workflows (specifically https://github.com/grafana/plugin-workflows/blob/master/release.yml)

1. Validation via pluginchecker
2. Version update via tag

**Comment by HadesArchitect** [comment]
Blocked by #36

**Comment by futuarmo** [comment]
We don't use Mage build system for the backend (I don't see any reason to use it, but I'll check docs again later), so release.yml file need to be configurated

**Comment by HadesArchitect** [comment]
I don't think it should anyhow affect backend, what I meant was more about signing plugin, rewriting versions etc. @futuarmo

**Comment by HadesArchitect** [comment]
So, the build now can be done without mage, because it's required to generate go build manifest :(
